recipes/wg21-linear_algebra/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Minor nitpick, but see https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/header_only/all/test_v1_package/CMakeLists.txt for a way to avoid having to duplicate C++ code for both test packages :)

Reply:
Nice, that certainly feels better than duplicating it.  That's done now.